.github/workflows/stale.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   stale:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/stale@v4
+      - uses: actions/stale@v4.1.0
```

Comment:
This actually isn't the latest version of `actions/stale`. The latest version is [v5.0.0](https://github.com/actions/stale/releases/tag/v5.0.0). I'm not really sure why dependabot hasn't tried to bump this one for us but I figured I'd leave it for a separate PR. The point of this PR is simply to correct the version numbers, not to change anything.